_posts/2024-04-01-openttd-14-0.md

```diff
@@ -0,0 +1,125 @@
+Do I dare to say: this has been the biggest release yet?
+
+<!-- more -->
+
+Sometimes people try to tell us OpenTTD is dead.
+Well, 14.0 shows you it really isn't.
+There are so many new goodies in 14.0, that I don't even know where to start!
+Let's go over a few, just to get you a feel for what to expect when you launch OpenTTD 14.0.
+
+Read more about our birthday celebration post [here](https://www.openttd.org/news/2024/03/06/happy-birthday).
+
+## When one day isn't a day
+
+It is now possible to make the calendar time walk slower, up to a point it is frozen.
+This doesn't influence vehicle movement at all, nor the amount of goods transported, town growth etc.
+But it does influence vehicle introduction dates, inflation, and aging.
+
+With this feature, we hope to enable different game-plays, where people want to have more time to build out their empire before vehicles expire, inflation gets too big, etc.
+
+A brand new day!
+For some a bit slower than others.
+
+Read more about this new feature [in the devblog](https://www.openttd.org/news/2024/03/23/timekeeping).
+
+## Ship Pathfinder
+
+The ship pathfinder got some great love.
+It is now capable of directing ships over vast distances without the need for buoys.
+In fact, one could say, you don't need buoys anymore at all.
+The ships will still find their way!
+
+Finally, no more "Ship is lost" non-sense.
+This enables ship-only games.
+
+Read more about this new feature [in the devblog](https://www.openttd.org/news/2024/02/24/new-ship-pathfinder).
+
+## Unbunching
+
+Do you also hate when busses start hugging each other on a route?
+And you so did your best to ensure you send them out the depot with a good distance of each other.
+But something must have happened down the road, and now they are very close to each other!
+
+Introducing: unbuncing.
+
+A new feature that allows you to tell all vehicles in a shared order pool to keep their distance; automatically, done for you.
+
+Read more about this new feature [in the devblog](https://www.openttd.org/news/2024/02/10/unbunching).
+
+## Social Platform integration
+
+OpenTTD 14.0 allows better integration with Social Platforms.
+This is totally optional, and has to be installed separately.
+But once done, OpenTTD can talk to platforms like Steam, Discord and GOG Galaxy.
+
+For now, the functionality is limited: friends can see you play OpenTTD, if you are in a server or not, and what kind of map you are playing.
+For future versions, we intend to expand on that, making more possible.
+But, small steps.
+
+Read more about this new feature [in the devblog](https://www.openttd.org/news/2024/03/02/social-integration).
+
+## Survey support
+
+To finally put a rest to endless debates about what setting is used and which is not, OpenTTD not has an opt-in survey tool built-in.
+Over time this will allow us, developer, to better understand how OpenTTD is played, so we can better anticipate what will be a hit or a miss.
```

Comment:
```suggestion
To finally put a rest to endless debates about what setting is used and which is not, OpenTTD now has an opt-in survey tool built-in.
Over time this will allow us, developers, to better understand how OpenTTD is played, so we can better anticipate what will be a hit or a miss.
```